gui/src/components/JobList.js

```diff
@@ -46,27 +47,66 @@ const renderBody = (jobs, fetching, error) => {
   }
 }
 
-export const JobList = ({jobs, fetching, error}) => (
-  <Card>
-    <Table>
-      <TableHead>
-        <TableRow>
-          <TableCell>ID</TableCell>
-          <TableCell>Created</TableCell>
-          <TableCell>Initiator</TableCell>
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        {renderBody(jobs, fetching, error)}
-      </TableBody>
-    </Table>
-  </Card>
-)
+const handleChangeRowsPerPage = () => {
+  console.log('handleChangeRowsPerPage')
+}
+
+export class JobList extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      page: 0
+    }
+    this.handleChangePage = this.handleChangePage.bind(this)
+  }
+
+  handleChangePage (e, page) {
+    const {fetchJobs, pageSize} = this.props
+
+    fetchJobs(page + 1, pageSize)
```

Comment:
Why don't we use the `next` HREF included in the paginated response? The next HREF is intended to be an opaque URI that you can be used to navigate through a collection that will include markers to prevent certain kinds of bugs, e.g: a filter to prevent new records from appearing during browsing through a collection.